File: Java/PTwPZ_Kalkulator.java
import java.util.Scanner;

class Main 
{
    public static void main(String[] args)
    {
        Scanner input = new Scanner(System.in);
        int N = Integer.parseInt(input.nextLine());

        for (int i = 0; i < N; i++) 
        {
            String data = input.nextLine();
            int result = 0;
            boolean add = false;
            boolean subtract = false;
            for (int j = 0; j < data.length(); j++) 
            {
                char symbol = data.charAt(j);
                if (symbol == '+') 
                {
                    add = true;
                    subtract = false;
                } 
                else if (symbol == '-') 
                {
                    subtract = true;
                    add = false;
                } else 
                {
                    int number = Character.getNumericValue(symbol);
                    if (add) 
                    {
                        result += number;
                        add = false;
                    } 
                    else if (subtract) 
                    {
                        result -= number;
                        subtract = false;
                    } 
                    else
                        result = number;
                }
            }
            System.out.println(result);
        }
        input.close();
    }
}

File: Python/PTwPZ Kalkulator.py
N = int(input())
for _ in range(N):
    data = input()
    result = 0
    add = False
    subtract = False
    for symbol in data:
        if symbol == '+':
            add = True
            subtract = False
        elif symbol == '-':
            subtract = True
            add = False
        else:
            if add:
                result += int(symbol)
                add = False
            elif subtract:
                result -= int(symbol)
                subtract = False
            else:
                result = int(symbol)
    print(result)